README.rst

```diff
@@ -18,13 +18,35 @@ Disclaimer
 
 .. code::
 
-    $ mkdir -p ~/work/tahoe-hawthorn  # It needs its own new directory
-    $ cd ~/work/tahoe-hawthorn
+    $ mkdir -p ~/work/tahoe-juniper  # It needs its own new directory
+    $ cd ~/work/tahoe-juniper
     $ git clone https://github.com/appsembler/devstack.git
-    $ cd devstack  # Now the `devstack` repo should be on the `hawthorn` branch
+    $ cd devstack  # Now the `devstack` repo should be on the `juniper` branch
+    $ git checkout juniper
+
+Create an ``options.local.mk`` file in your local ``devstack`` directory:
+
+.. code::
+
+  OPENEDX_RELEASE=juniper.master
+  COMPOSE_PROJECT_NAME=junstack
+  DEFAULT_SERVICES=lms+studio+forum+amc+amc-frontend
+  DB_SERVICES=lms studio amc
+
+Now provision your devstack:
+
+.. code::
+
     $ make dev.provision
+
+If all goes well, you can then start the devstack:
+
+.. code::
+
     $ make dev.up
 
+One known issue is that theming does not get installed with the ``make dev.provision`` step. See the "Theme" section below for troubleshooting.
+
```

Comment:
Since we don't update this document often, it would not be a good place to track known issues. Let's work on reproducing the issue first before committing it as a long-term known issue.

```suggestion
```